addons/planner/static/src/js/planner.js

```diff
@@ -284,6 +285,9 @@
             }
             self.update_planner(page_id);
         },
+        on_close:function(){
```

Comment:
If necessary, those changes should be done in a specific commit / branch. What is the related bug / feature ?